esmvaltool/utils/recipe_test_workflow/recipe_test_workflow/rose-suite.info

```diff
@@ -1,3 +1,4 @@
 owner=ESMValTool community
 project=RTW
 title=The Recipe Test Workflow (RTW)
+description=This is currently a prototype
```

Comment:
I'm going to suggest removing this for now in favour of adding a link to the documentation once it is fully integrated into the ESMValTool documentation.
```suggestion
```

Reply:
Can we keep it until we have the link to the documentation when it is integrated? Should we create an issue for this?

Reply:
If you want to keep it then I would recommend updating the description to match what is currently in the [RTW documentation](https://github.com/ESMValGroup/ESMValTool/blob/recipe_test_workflow_prototype/esmvaltool/utils/recipe_test_workflow/doc/source/about.rst):
```suggestion
description=The RTW is a Cylc 8 workflow that is used to regularly run recipes so
           =issues can be discovered during the development process sooner
           =rather than later
```

Reply:
> Should we create an issue for this?

Yes, thank you for the reminder! See #3253 :)